src/features/query-bar/__tests__/TemplateButton-test.js

```diff
@@ -81,5 +81,16 @@ describe('feature/query-bar/components/TemplateButton', () => {
             const entryButtonWrapper = shallow(wrapper.instance().renderEntryButton());
             expect(entryButtonWrapper.props().className.includes(activeTemplateClassName)).toEqual(expectedReturn);
         });
+
+        test.each`
+            templates | expectedMessage             | description
+            ${null}   | ${'Template Name'}          | ${'Should render templates loading message'}
```

Comment:
Is this accurate? -> "Should render templates loading message"

Reply:
So the text on the button while the templates are loading is "Template Name" which is technically the templates loading message.

Reply:
Does it mean loading message? I'm not sure about that. If I see the word "loading message", I expect a spinner or similar.

My understanding is that "Template Name" is a placeholder value. Here's the definition of HTML "placeholder": The placeholder attribute specifies a short hint that describes the expected value of an input field (e.g. a sample value or a short description of the expected format).